msteams-platform/tabs/how-to/app-initialization-process.md

```diff
@@ -0,0 +1,91 @@
+---
+title: App Initialization Process for Tab Apps
+description: Learn about the app initialization process, including the APIs involved and recommendations for their use in code.
+author: JoshuaPartlow
+ms.author: joshuapa
+ms.service: msteams
+ms.topic: concept-article #Required; leave this attribute/value as-is.
+ms.date: 03/16/2025
+
+#CustomerIntent: As a developer, I want to understand how the app initialization process works for tab apps so that my app can make the appropriate API calls and work as recommended.
+---
+
+# App initialization process
+
+App initialization is a foundational aspect of authoring a tab app and ensures that apps work as expected. All apps created for Microsoft Teams that use TeamsJS must be initialized. The process of app initialization is a finite state machine and it uses APIs with specific roles to advance and monitor the event flow. The following sections will:
```

Comment:
I am not sure if we should say `All apps ··· using TeamsJS must be initialized`. There are some reasons on my concerns.

 1. There are some apps already who don't do initialization.
 2. Tree shaking and some capabilities provide ability to use TeamsJS without initialization on some functionality.
 
 I would like to know what other people think of this. Maybe consult Saurab, and Jeff as TeamsJs owner?

Reply:
Aligned with our separate discussion in latest rev.